ims-react/src/containers/customers/newInquiry/components/InquiryForm.js

```diff
@@ -127,36 +128,47 @@ const newUserForm = () => (
   </Container>
 );
 
-const findExistingUserForm = () => (
-  <Container>
-    <h3>Find existing user</h3>
-    {/* <SelectableSearchCustomers></SelectableSearchCustomers> */}
-  </Container>
-);
+const onCustomerSelect = (firstName, lastName, id, formikProps) => {
+  formikProps.setFieldValue('selectedCustomer.id', id)
+  formikProps.setFieldValue('selectedCustomer.firstName', firstName)
+  formikProps.setFieldValue('selectedCustomer.lastName', lastName)
+}
 
-const selectCustomer = (referenceData, formikProps) => (
+const selectCustomer = (referenceData, formikProps, customerSearchStore) => (
   <CwigCard>
     <Row>
       <h3>Select Customer</h3>
     </Row>
-    <Row>
-      <RadioButtonGroup 
-        id="userType"
-        options={["Anonymous", "Create New Customer", "Add Existing Customer"]}
-        value={formikProps.usertype}
-      />
-    </Row>
+    { formikProps.values.selectedCustomer.id === '' ?
```

Comment:
This is somewhat complicated logic that hides/shows various component of the AddCustomer section on the new inquiry form. I think I'll do a full clean up refactor when I have the entire page working properly.  I will probably pull more components out, as well.